packages/xo-web/src/xo-app/vm-import/index.js

```diff
@@ -188,17 +190,26 @@ const parseFile = async (file, type, func) => {
   }
 }
 
-const getRedirectionUrl = vms =>
-  vms.length === 0
+const getRedirectionUrl = vms => {
```

Comment:
You can modify `_importVmFromUrl` to return an array instead of modifying this method.